go.mod

```diff
@@ -13,14 +13,15 @@ require (
 	github.com/fluxcd/notification-controller/api v0.19.0
 	github.com/fluxcd/pkg/apis/meta v0.11.0-rc.1
 	github.com/fluxcd/pkg/runtime v0.13.0-rc.5
+	github.com/fluxcd/pkg/ssa v0.3.1
+	github.com/fluxcd/source-controller/api v0.19.0
```

Comment:
We can’t make NC depend on any other Flux API. If we need to operate on sources we ca use unstructured objects.